en/sql/catalog.rst

```diff
@@ -984,6 +1025,7 @@ The following example shows how to retrieve system classes that can be accessed
       'db_root'
```

Comment:
'db_ha_apply_info' class need to be added. 
I think it will be covered in another issue.